piksi_tools/console/console.py

```diff
@@ -25,6 +25,7 @@
 
 # Shut chaco up for now
 import warnings
+import time
```

Comment:
Doesn't look like time is used here? Were we going to automatically add a time suffix the to the filename?